test/build/bundle_test.js

```diff
@@ -97,6 +97,18 @@ suite('Bundler', () => {
     return link != null;
   };
 
+  suite('urlFromPath()', () => {
+
+    test.skip('creates a URL path relative to root when called with a file path', () => setupTest({
```

Comment:
rather than always skipping the test, can you conditionally skip it based on the platform?